### PR TITLE
LRCA-124 - Fixed validations removing data

### PR DESCRIPTION
**Pull request by ajwhitley**
Provider information was being removed if the json object was not valid.
Changed this to include the provider information when saving json input.
Also added validations to check json to make sure it has 2 properties, or mark invalid.

### CODE REVIEW

File: src/extensions/FormBuilder/hooks/useFormConfig.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function useFormConfig(handleFieldChange, initialState = {}) {
   const formConfig = useProviderConfig(handleFieldChange, initialState);
 
   const loadState = ({ steps = [], provider = {} }) => {
+    console.log('loadState', { steps, provider });
```
Comment (camiloriosf) [inline src/extensions/FormBuilder/hooks/useFormConfig.js:17]: let's remove this console.log

File: src/extensions/FormBuilder/hooks/useProviderConfig.js [hunk 2]
```diff
@@ -53,10 +53,11 @@ export default function useProviderConfig(onChangeField, { provider = {} } = {})
     },
 
     update: ({ parameters: newParameters, type: newType }) =>
+      console.log({ newParameters, newType, values }) ||
```
Comment (camiloriosf) [inline src/extensions/FormBuilder/hooks/useProviderConfig.js:56]: let's remove this console.log

**Review by camiloriosf — CHANGES_REQUESTED** [review]

**Review by camiloriosf — APPROVED** [review]
LGTM

**Review by maxtechera — DISMISSED** [review]
LGTM!

**Review by camiloriosf — APPROVED** [review]

**Pull request merged**